### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by JAVGan**
This PR includes changes in the documentation to make it aligned with the most recent changes.

Commits:

- [Refactor the documentation structure](https://github.com/release-engineering/pubtools-marketplacesvm/commit/308ad4895e8da2932e9d6b020d7c262c185259b4)
-  [Add common docs to the index](https://github.com/release-engineering/pubtools-marketplacesvm/commit/56058c0c2011d50721edd2931ddb59319d1c213e)
- [Update the credentials section](https://github.com/release-engineering/pubtools-marketplacesvm/commit/f121ab4bcc6b97ea080b0bc14caa3ae3ac71b024)
- [Update the mappings section](https://github.com/release-engineering/pubtools-marketplacesvm/commit/a0d67b90287192e97f2d1d7aefd7cf4b898e5125)
- [Update the entrypoints documentation](https://github.com/release-engineering/pubtools-marketplacesvm/pull/22/commits/68f7843bc17b7cba9b6e4b001d94a122c09ceaa8)
- [Update the copyright year](https://github.com/release-engineering/pubtools-marketplacesvm/pull/22/commits/cbb59f1c94a737584d107864f8df9b0168c1c1cc)

Refers to SPSTRAT-327

### CODE REVIEW

File: docs/common/credentials.rst [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+- ``AWS_IMAGE_ACCESS_KEY``: The AWS access key for uploading the images and import them as AMIs
+- ``AWS_IMAGE_SECRET_ACCESS``: The AWS secret key for uploading the images and import them as AMIs
+- ``AWS_MARKETPLACE_ACCESS_KEY``: The AWS access key for publishing a new product version with its AMI into the marketplace
+- ``AWS_MARKETPLACE_SECRET_ACCESS``: The AWS secret key for publishing a new product version with its AMI into the marketplace
+- ``AWS_ACCESS_ROLE_ARN``: The AWS role ARN for publishing a new product version with its AMI into the marketplace
+- ``AWS_GROUPS``: The default sharing accounts for uploading and importing AMIs
+- ``AWS_SNAPSHOT_ACCOUNTS``: The default sharing snapshot accounts for uploading and importing AMIs
+- ``AWS_REGION``: The default region for uploading and importing AMIs
+- ``AWS_S3_BUCKET``: The S3 bucket name to upload the raw VM images
```
Comment (JAVGan) [inline docs/common/credentials.rst:60]: @jajreidy please let me know if the description for each of these are correct.. I did them based on what I remember

**Comment by JAVGan** [comment]
@lslebodn @jajreidy I highly recommend you to pull this PR locally, builds the docs and check them with your browser

```
gh pr checkout 22
tox -e docs
[google-chrome|firefox|...] docs/_build/html/index.html
```

**Review by jajreidy — CHANGES_REQUESTED** [review]

File: docs/common/credentials.rst [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+**AWS Format**
+
+The following properties are supported for AWS ``auth``:
+
+- ``AWS_IMAGE_ACCESS_KEY``: The AWS access key for uploading the images and import them as AMIs
+- ``AWS_IMAGE_SECRET_ACCESS``: The AWS secret key for uploading the images and import them as AMIs
+- ``AWS_MARKETPLACE_ACCESS_KEY``: The AWS access key for publishing a new product version with its AMI into the marketplace
+- ``AWS_MARKETPLACE_SECRET_ACCESS``: The AWS secret key for publishing a new product version with its AMI into the marketplace
+- ``AWS_ACCESS_ROLE_ARN``: The AWS role ARN for publishing a new product version with its AMI into the marketplace
```
[the remark this answers is not in the document]
Comment (JAVGan) [reply docs/common/credentials.rst:56]: Is the role for accessing the image to scan as well? Or to publish?

And just to be sure, the scan images are the `_MARKETPLACE_*` ones, right?

[the remark this answers is not in the document]
Comment (JAVGan) [reply docs/common/credentials.rst:56]: Does this make it correct?

```
- ``AWS_MARKETPLACE_ACCESS_KEY``: The AWS access key for for scanning the AMI for vulnerabilities once uploaded
- ``AWS_MARKETPLACE_SECRET_ACCESS``: The AWS secret key for scanning the AMI for vulnerabilities once uploaded
- ``AWS_ACCESS_ROLE_ARN``: The AWS role ARN for accessing the AMI for publishing it into the marketplace
```

[the remark this answers is not in the document]
Comment (jajreidy) [reply docs/common/credentials.rst:56]: Here's what it says:

AccessRoleArn (string) – IAM role Amazon Resource Name (ARN) used by AWS Marketplace to access the provided AMI. For details about creating and using this ARN, see [Giving AWS Marketplace access to your AMI](https://docs.aws.amazon.com/marketplace/latest/userguide/ami-single-ami-products.html#single-ami-marketplace-ami-access) in the AWS Marketplace Seller Guide.


Located here: https://docs.aws.amazon.com/marketplace-catalog/latest/api-reference/ami-products.html#working-with-single-AMI-products

[the remark this answers is not in the document]
Comment (JAVGan) [reply docs/common/credentials.rst]: Added on 246fcb28fdea00715a1947a22993f21d2a1f0fe5

**Review by jajreidy — APPROVED** [review]
LGTM

**Pull request merged**